pandas/sparse/array.py

```diff
@@ -503,6 +505,34 @@ def mean(self, axis=None, dtype=None, out=None):
             nsparse = self.sp_index.ngaps
             return (sp_sum + self.fill_value * nsparse) / (ct + nsparse)
 
+    def value_counts(self, dropna=True):
```

Comment:
so it makes sense to define ``.value_counts`` in ``tseries/base.py`` to handle the datetimelikes (and the reconstruction)